JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -363,5 +363,16 @@ module Microsoft.ApplicationInsights {
             
             return true;
         }
+
+        public static getAbsoluteUrl = (function () {
+            var a;
+
+            return function (url) {
+                if (!a) a = document.createElement('a');
```

Comment:
can you pls explain why it doesn't affect minimization? Does minifier creates global `document` object to use everywhere?